app/src/main/java/com/android/unio/ui/authentication/AccountDetails.kt

```diff
@@ -340,7 +355,20 @@ private fun UserTextFields(
           onLastNameChange(it)
         }
       },
-      value = lastName)
+      value = lastName,
+      trailingIcon = {
+        if (lastName.isNotEmpty()) {
```

Comment:
Instead of hiding the button if there is no text, consider maybe simply disabling the button, that way there is no big UI difference

Reply:
Yeah I agree, it is best to let the user know the feature (button) exists before they event type anything. I'll do that before merging.